src/Components/Forms/TestsAndTreatments.js

```diff
@@ -0,0 +1,26 @@
+//TestsAndTreatment
+
+import { connect } from 'react-redux';
+import React from 'react';
+
+const TestsAndTreatment = ({
+  patient,
+  encounter,
+  formatDate,
+  languageDirection,
+  history,
+  verticalName,
+}) => {
+  return <React.Fragment>TestsAndTretments</React.Fragment>;
+};
+
+const mapStateToProps = (state) => {
+  return {
+    patient: state.active.activePatient,
```

Comment:
as the comment in the DiagnosisAndRecommendations component

Reply:
not relevant